server/autotest_server/__init__.py

```diff
@@ -402,6 +378,7 @@ def update_test_settings(user, settings_id, test_settings, file_url):
         os.chmod(TEST_SCRIPT_DIR, 0o755)
 
         files_dir = os.path.join(settings_dir, "files")
+        test_settings["_files"] = files_dir
```

Comment:
I don't think this change is related to your work on this PR, and should be pulled out

Reply:
This is coming from #593. I branched off of that, because when I began working on this PR, #593 has not been merged. I think the duplicated commits should disappear when this branch is merged. I think I could also rebase to fix it?